config/initializers/assets.rb

```diff
@@ -10,3 +10,5 @@
   'node_modules/@18f/identity-design-system/dist/assets/img',
   'node_modules/@18f/identity-design-system/dist/assets/fonts',
 )
+
+require Rails.root.join('lib', 'extensions', 'propshaft', 'asset')
```

Comment:
is `lib` on the load path by this time? if so, we can do
```suggestion
require 'extensions/propshaft/asset'
```
also I'd consider moving the require up top, that's usually an easier spot to find them

Reply:
Yes and yes! Updated in a0acea1.